repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,2836 @@
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: decontamination_services_management
+          locale.en.name: Decontamination services management
+        /design_and_engineering:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: design_and_engineering
+          locale.en.name: Design and engineering
+        /estates_manager:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: estates_manager
+          locale.en.name: Estates manager
+        /facilities_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: facilities_management
+          locale.en.name: Facilities management
+        /finance_manager:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: finance_manager
+          locale.en.name: Finance manager
+        /hotel_services_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: hotel_services_management
+          locale.en.name: Hotel services management
+        /human_resources__hr__manager:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: human_resources__hr__manager
+          locale.en.name: Human resources (HR) manager
+        /integrated_urgent_care_nhs_111_team_leader:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: integrated_urgent_care_nhs_111_team_leader
+          locale.en.name: Integrated urgent care/NHS 111 team leader
+        /performance_and_quality_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: performance_and_quality_management
+          locale.en.name: Performance and quality management
+        /practice_manager:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: practice_manager
+          locale.en.name: Practice manager
+        /project_management_and_procurement:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: project_management_and_procurement
+          locale.en.name: Project management and procurement
+        /project_manager:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: project_manager
+          locale.en.name: Project manager
+        /purchasing_and_contract_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: purchasing_and_contract_management
+          locale.en.name: Purchasing and contract management
+        /strategic_management:
+          jcr:primaryType: hippotaxonomy:category
+          hippotaxonomy:key: strategic_management
+          locale.en.name: Strategic management
+      /medical_associate_professionals:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: medical_associate_professionals
+        locale.en.name: Medical associate professionals
+      /medical_doctors:
```

Comment:
Sub professions `Haematology (doctor)`, `Immunology` & `Intensive care medicine` seems to be missing under the `Medical doctors` profession. Could we add them ?

Reply:
Have added these now